lib/generators/solidus_braintree/install/templates/app/helpers/solidus_braintree/braintree_checkout_helper.rb

```diff
@@ -11,6 +11,8 @@ def braintree_3ds_options_for(order)
         onLookupComplete: nil, # populated after tokenization
         amount: order.total,
         email: order.email,
+        collectDeviceData: true,
+        IpAddress: request.remote_ip,
```

Comment:
This is clearly a GDPR violation. We need this to be configurable, so folks can opt out.